src/commands/outputMetrics.ts

```diff
@@ -11,6 +11,12 @@ interface OutputNode {
     };
 }
 
+interface OutputInfoNode {
```

Comment:
i recommend to rename OutputInfoNode to OutputUnknownFile.

Reply:
clarified